tests/system/model_monitoring/test_app.py

```diff
@@ -371,3 +373,54 @@ def test_inference_feature_set(self) -> None:
         time.sleep(2.4 * self.app_interval_seconds)
 
         self._test_v3io_records(self.endpoint_id)
+
+
+@TestMLRunSystem.skip_test_if_env_not_configured
+@pytest.mark.enterprise
+class TestModelMonitoringInitialize(TestMLRunSystem, _V3IORecordsChecker):
```

Comment:
You don't need the second inheritance:
```suggestion
class TestModelMonitoringInitialize(TestMLRunSystem):
```